src/main/java/org/sonarsource/sonarlint/ls/settings/SettingsManager.java

```diff
@@ -158,7 +160,11 @@ private void notifyListeners(WorkspaceSettings newWorkspaceSettings, WorkspaceSe
 
   // Visible for testing
   CompletableFuture<Map<String, Object>> requestSonarLintConfigurationAsync(@Nullable URI uri) {
-    LOG.debug("Fetching configuration for folder '{}'", uri);
+    if (uri != null) {
```

Comment:
Fix for the logging. Previously this line just silently skipped logging in case uri is null.